app/utils/errors.ts

```diff
@@ -70,15 +70,15 @@ export const getFullErrorMessage = (error: unknown, intl?: IntlShape, depth = 0)
         return `${message}; ${getFullErrorMessage(error.details, intl, depth + 1)}`;
     }
 
-    return message;
+    return message!;
```

Comment:
What is causing us to need this? Same on line 81. We should add some default value instead of using the `!`.